shristy-kumari/html-css-task/style.css

```diff
@@ -0,0 +1,490 @@
+}
+table{
+    width: 100%;
+    background-color: #edf3fc;
+    border-collapse: collapse;
+}
+
+th{
+    background-color: #182059;
+    color: #fff;
+    border: none;
+}
+table th,td{
+    padding: 13px 20px;
+    border-bottom: 1px solid #d9d9d9;
+    text-align: center;
+}
+.fa-check,.fa-edit{
+    color: green;
+}
+.fa-close,.fa-trash-alt{
+    color: red;
+}
+.table-header{
+    margin: 20px 5px 10px;
+    display: flex;
+    justify-content: space-between;
+    padding: 10px 20px;
+    text-align: center;
+}
+.table-header h1{
+    font-size: 18px;
+    font-weight: 400; 
+}
+.btns button{
+    background-color: #182059;
+    color: #fff;
+    padding: 10px 20px;
+    border: none;
+    border-radius: 5px;
+    cursor: pointer;
+}
+
+/* tab styling */
+input[type="radio"] {
+    display: none;
+}
+
+.add-record{
+    text-decoration: none;
+    background-color: #182059;
+    color: #fff;
+    padding: 10px 20px;
+    border: none;
+    border-radius: 5px;
+    cursor: pointer;
+    font-size: 16px;
+    display: inline-block;
+    float: right;
+    margin: 10px 20px;
+}
+
+.label {
+    padding: 10px 20px;
+    background-color: #fff;
+    color: #000;
+    cursor: pointer;
+    transition: .4s;
+    float: right;
+    margin: 10px 0px;
+}
+
+
+.tab-content {
+    display: none;
+    margin: 20px 0;
+}
+
+input[type="radio"]:checked+.label {
+    background-color: #182059;
+    color: #fff;
+}
+
+#tab1:checked~#content1,
+#tab2:checked~#content2{
+    display: block;
+}
+.pagination{
+    display: flex;
+    justify-content: center;
+    gap: 10px;
+    padding:30px 0px  10px ;
+}
+
+.pagination a{
+    text-decoration: none;
+    display: flex;
+    justify-content: center;
+    text-align: center;
+    background-color: #fff;
+    padding: 7px 20px;
+    border-radius: 10px;
+}
+.pagination button{
+    background-color: #42456a;
+    color: #fff;
+    padding: 10px 20px;
+    border: none;
+    border-radius: 5px;
+    cursor: pointer;
+} 
+
+/* product card styling here */
+
+.productscard-container{
+    width: 100%;
+    border-top: 1px solid #cfcfcf;
+}
+
+.productscard-container{
+    width: 90%;
+    display: flex;
+    flex-wrap: wrap;
+    gap: 20px;
+    margin: 0px auto;
+    padding-top: 10px;
+}
+
+.product-card{
+    width: 260px;
+    height: 200px;
+    background-color: #fff;
+    border-radius: 10px;
+    display: flex;
+    padding: 0px  10px;
+    color: #4b4a4a;
+    font-weight: 400;
+    margin: 10px 0px;
+}
+.product-text{
+    width: 100%;
+    padding: 10px 8px;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+}
+.product-name{
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 8px;
+}
+.product-info{
+    display: flex;
+    justify-content: space-between;
+}
+.left div,.right div{
+    margin-bottom: 8px;
+    font-size: 13px;
+}
+.head{
+    color: rgb(151, 150, 150);
+}
+
+/* sidebar styling here */
+
+#available,#notAvailable {
```

Comment:
Is this required? I didn't understand the purpose. The naming convention is also inconsistent.